pkg/config/user_config.go

```diff
@@ -350,6 +351,7 @@ func GetDefaultConfig() *UserConfig {
 			ShowFileTree:             true,
 			ShowRandomTip:            true,
 			CommandLogSize:           8,
+			GoneBranchText:           "↑gone",
```

Comment:
I don't think this needs to be configurable. I'd just hard-code it (but in `pkg/i18n/english.go`) and we'll see if anybody actually has a use case for customising it. The less custom stuff we need to support the better.

Reply:
Yeah originally I thought some people might want to set it to `""` so that nothing is shown, and the presentation of this would look like this is completely a local branch.